pages/pipelines/functions.rst

```diff
@@ -475,11 +477,17 @@ processed for this message.
 
 create_message
 --------------
-``create_message([message: string], [source: string], [timestamp: DateTime])``
+``create_message([message: Message], [source: string], [timestamp: DateTime])``
 
 Creates a new message with from the given parameters. If any of them is omitted, its value is taken from the currently
 processed message. If ``timestamp`` is omitted, the timestamp of the created message will be the timestamp at that moment.
 
+clone_message
+-------------
+``clone_message([message: Message])``
+
+Clones a message. If ``message`` is omitted, this function uses the currently processed message.
```

Comment:
see above it's a string, in case when it's omitted the `message` field of the currently processed message is used.